tests/js/JSStandardBearTest.py

```diff
@@ -77,3 +95,31 @@
                                                       bad_file_undef,
                                                       bad_file_ifelse,
                                                       bad_file_func_name,))
+
+
+jsstandardBearTsTest = verify_local_bear(
```

Comment:
Please adhere to the correct casing + Don't shorten parts of the name ;) Ts --> Typescript